examples/fedpca_examples/perform_pca/README.md

```diff
@@ -0,0 +1,41 @@
+# Federated Principal Component Analysis Example
+This example performs federated principal component analysis. The goal is to compute the principal components of a subset of the MNIST dataset, under the condition that the data is distributed across two distinct clients.
+
+This is achieved by each client performing PCA locally at first, then the principal components are sent to a central server to be merged.
+
+Each client performs Dirichlet subsampling on the whole dataset to produce heterogeneous local datasets. This is done to ensure that local principal compoents are distinct enough across different clients.
```

Comment:
I would just say "distinct" instead of "distinct enough." We don't really define what we mean by "enough" in this instance